test/Microsoft.AspNetCore.Mvc.Cors.Test/project.json

```diff
@@ -5,11 +5,9 @@
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Http": "1.2.0-*",
-    "Microsoft.AspNetCore.Mvc": "1.2.0-*",
```

Comment:
Nice this mondo dependency can be removed in this test project. Have you tried all the rest?